### PR TITLE
Readme improvements

### PR DESCRIPTION
**Pull request by NickVolynkin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,23 +28,34 @@ Pre-Requisites :
...
 * m2eclipse-tycho 0.6 or later
 
-How to use
+Accessing the archetype from Eclipse
 -------------------
-
 In Eclipse, first add the Open Archetypes catalog :
 
-* On the Archetypes Preferences page (Window > Preferences > Maven > Archetypes), click on the "Add Remote Catalog..." button
+1. On the Archetypes Preferences page (Windows: `Window` > `Preferences`; OS X: `⌘,`), open `Maven` > `Archetypes`, click on the `Add Remote Catalog...` button
```
Comment (fbricon) [inline README.md]: OS X: `Eclipse` > `Preferences` or `⌘,`

File: README.md [hunk 2]
```diff
@@ -31,23 +28,34 @@ Pre-Requisites :
...
 
     - Catalog file : http://open-archetypes.github.com/maven-repo/releases/
-    - Description : Open Archetypes
+    - Description : "Open Archetypes"
```
Comment (fbricon) [inline README.md]: That might be confusing as you don't need to type the " in the wizard. 
If formatting is what you're after, then use `Open Archetypes`

Comment (NickVolynkin) [inline README.md]: Oh, you're right about confusing ""s.

Comment (NickVolynkin) [inline README.md]: Should I now re-initiate a pull request with the new changes?

Comment (fbricon) [inline README.md]: no, just push -f to your branch

**Comment by NickVolynkin** [comment]
Fixed.

**Comment by fbricon** [comment]
Thanks for the contribution guys

**Comment by NickVolynkin** [comment]
Thanks for your great archetype, Fred! It saved me a couple of days for setting up Eclipse.

**Pull request merged**